samples/java/semantickernel-samples/src/main/java/com/microsoft/semantickernel/Config.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.semantickernel;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Properties;
+
+public class Config {
+  public static String getOpenAIKey(String conf) {
+    return getConfigValue(conf, "token");
```

Comment:
I believe it is the way it is so that an application may as well have two or more instances of the Semantic Kernel, one for each subscription (in the case of multiple subs). A customer may as well have one instance of SK for Azure OpenAI, and one for OpenAI.

Unless I misunderstood your comment about 'constants'.